### DISCUSSION TITLE
Invert screen color possible?

### DISCUSSION THREAD

**Discussion by kagehisa** [jayofelony/pwnagotchi#4] [General]
After studying the PR's for pwnagotchi-torch I stumpled upen the setting `ui.invert`. 
I tried to use it with my waveshare display but it doesn't seem to do anything.
Here are the relevant lines from my config.
```
ui.display.enabled = true
ui.display.type = "waveshare_4"
ui.display.color = "black"
ui.display.rotation = 180

ui.invert = true
```

 Is it even doing what I'm expecting it to do? Or do I still need to maually exchange de Black and White defined values in view.py?

**Comment by jayofelony** [comment]
Yes, inverting does not work. Should be removed in 2.6.4 and up.

**Reply by kagehisa** [reply]
Do you perhaps have an idea as to how I can make this setting update save? Or could you point me to the setion of code where the config.toml gets parsed? Python is not my strong suit but I'm in the mood to try something^^

**Reply by jayofelony** [reply]
No just edit manually each update, or get used to dark mode hahaha

**Reply by kagehisa** [reply]
Ah the pain of manual labor... second update I really need to come up with something...
